registry.json

```diff
@@ -5547,5 +5547,19 @@
    "telegram": "THEBMOREJOKER",
    "operatorName": "Lee",
    "github": "THEBMOREJOKER"
+  },
+  {
+  "ticker": "CDRAG",
+   "poolId": "pool1wehqlvg34f3cpn48fa93qhnfkmxf5v3lxk9ymzukrm3tgxq34z2",
```

Comment:
Pool-Id for CDRAG is 766e0fb111aa6380cea74f4b105e69b6cc9a323f358a4d8b961ee2b4

Reply:
updated poolID thank you